### PR TITLE
feat: error message for invalid variables in custom event name

### LINKED ISSUE #7305: [CAL-1144] Custom event type name: Differentiate b/w valid & invalid variable
* Differentiate b/w an invalid variable and a valid variable in cases where variable value is decided during booking
* Also show "John Doe" for the preview of the {Scheduler} variable.

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1144](https://linear.app/calcom/issue/CAL-1144/custom-event-type-name-differentiate-bw-valid-and-invalid-variable)</sub>

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

Fixes #7305 

[Screencast 2023-02-27 23:52:33.webm](https://user-images.githubusercontent.com/84864519/221651579-cc453c1e-1761-4a29-956c-cd2e42d5dad3.webm)

**Environment**: Staging(main branch) / Production

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: apps/web/components/eventtype/CustomEventTypeModal.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+          style={{
+            backgroundImage: "url(/calendar-preview.svg)",
+          }}>
+          <div className="m-auto flex items-center justify-center self-stretch">
+            <div className="mt-3 ml-11 box-border h-[110px] w-[120px] flex-col items-start gap-1 rounded-md border border-solid border-black bg-gray-100 text-[12px] leading-3">
+              <p className="overflow-hidden text-ellipsis p-1.5 font-medium text-gray-900">{previewText}</p>
+              <p className="ml-1.5 text-[10px] font-normal text-gray-600">8 - 10 AM</p>
+            </div>
+          </div>
+        </div>
```
Comment (roae) [inline apps/web/components/eventtype/CustomEventTypeModal.tsx:91]: Is It possible to avoid using arbitrary values here?
cc @sean-brydon

**Review by roae — APPROVED** [review]
looks good overall to me

**Review by CarinaWolli — APPROVED** [review]
Built #7454 on top of your branch and everything worked smoothly 🙌🏻

**Comment by CarinaWolli** [comment]
@G3root can you solve the merge conflicts please

**Comment by G3root** [comment]
@CarinaWolli fixed the merge conflict but the linting is failing for whatever reason 🤔

**Pull request merged**